tools/configure_esp_secure_cert.py

```diff
@@ -425,6 +548,11 @@ def main():
         dest='summary',action='store_true',
         help='Provide this option to print efuse summary of the chip')
 
+    parser.add_argument(
+        '--configure_ds',
```

Comment:
Can you also add this line to the commands for configuring DS peripheral in the [tools/README](https://github.com/espressif/esp_secure_cert_mgr/tree/main/tools#readme)


Also, the [main README](https://github.com/espressif/esp_secure_cert_mgr#readme) needs to be updated with the new name for the script.